sql/world/base/add_default_simpleitemconfig.sql

```diff
@@ -0,0 +1,986 @@
+TRUNCATE TABLE mod_auctionhousebot_simpleitemconfig;
+INSERT INTO mod_auctionhousebot_simpleitemconfig (itemID, numStacks, stackSize)
+VALUES
```

Comment:
I used a web scraper on a Wotlk DB to help populate this file, but there was still a decent bit of manual work.